specification/applicationinsights/resource-manager/readme.md

```diff
@@ -435,7 +440,6 @@ input-file:
   - $(this-folder)/Microsoft.Insights/preview/2018-06-17-preview/workbooks_API.json
   - $(this-folder)/Microsoft.Insights/preview/2019-10-17-preview/workbookTemplates_API.json
   - $(this-folder)/Microsoft.Insights/preview/2018-05-01/componentProactiveDetection_API.json
-  - $(this-folder)/Microsoft.Insights/preview/2018-05-01/components_API.json
```

Comment:
Has this confirmed with the service team?

Reply:
mistake and has been solved